### PR TITLE
[Bug] Audit logs for compaction

### PR DESCRIPTION
**Pull request by ravikiran23**
Need to add audit logs for compaction process. 
Handling both the success and failure cases.

### CODE REVIEW

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala [hunk 1]
```diff
@@ -455,6 +456,8 @@ object CarbonDataRDDFactory extends Logging {
       logger
         .info("compaction need status is " + CarbonDataMergerUtil.checkIfAutoLoadMergingRequired())
       if (CarbonDataMergerUtil.checkIfAutoLoadMergingRequired()) {
+
+        logger.audit("Compaction request received.")
```
Comment (gvramana) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala]: add schema name and table name to log

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/Compactor.scala [hunk 2]
```diff
@@ -107,5 +107,8 @@ object Compactor {
         )
       logger.audit("Compaction request completed.")
     }
+    else {
+      logger.audit("Compaction request is failed.")
```
Comment (gvramana) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/Compactor.scala]: add logger.error also

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala (hunk 1 above)
Comment (ravikiran23) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/CarbonDataRDDFactory.scala]: fixed

File: integration/spark/src/main/scala/org/carbondata/spark/rdd/Compactor.scala (hunk 2 above)
Comment (ravikiran23) [inline integration/spark/src/main/scala/org/carbondata/spark/rdd/Compactor.scala]: fixed

**Comment by ravikiran23** [comment]
retest this please

**Pull request merged**